### PR TITLE
12.5 wrong button behaviour for cancelation pop up of a submission without consent

### PR DESCRIPTION
**Pull request by freshking**
## Description

Switch popup button behavior and change popup title and popup button text

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4930

## Screenshots

IMG_2673

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 1]
```diff
@@ -566,17 +566,17 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
...
 		)
 
 		alert.addAction(
 			UIAlertAction(
 				title: AppStrings.ExposureSubmissionTestResultAvailable.closeAlertButtonContinue,
-				style: .default
+				style: .default,
+				handler: { [weak self] _ in
+					self?.dismiss()
+				}
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:579]: This change does not make any sense to me. If you tab "Weiter", we cancel? And if you tap "Abbrechen", we continue? I don't think this is correct.

**Review by nickguendling — DISMISSED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift (hunk 1 above)
Comment (freshking) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:579]: @nickguendling This what is specified in the ticket. The popup says you are about to cancel. Do you want to cancel [Abbrechen] or continue [Weiter] with the cancellation.

Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:579]: I have contacted the PO/UX/UA directly to clarify this, it's been a long discussion already 😅

Comment (whiskey) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift]: "Weiter" is a terrible and ambiguous term in such a situation. This should be changed and I hope the discussion will solve that.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -634,17 +634,15 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "ExposureSubmissionTestresultAvailable_primaryButtonTitle" = "Weiter";
 
-"ExposureSubmissionTestresultAvailable_CloseAlertTitle" = "Achtung:\n Sie sind dabei den Vorgang abzubrechen. Sie haben Ihre Mitmenschen noch nicht gewarnt. Bitte schützen Sie Ihre Mitmenschen und schließen Sie den Vorgang ab.";
-
-"ExposureSubmissionTestresultAvailable_CloseAlertMessage" = "Möchten Sie den Vorgang wirklich abbrechen?";
+"ExposureSubmissionTestresultAvailable_CloseAlertTitle" = "Erst wenn Sie sich über Ihr Testergebnis informiert haben, können Sie bei Bedarf Ihre Mitmenschen warnen und so helfen, Infektionsketten zu unterbrechen.";
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:637]: As mentioned before I think this should be switched with the message. This looks weird. Just mentioning…

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -634,17 +634,15 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
 
 "ExposureSubmissionTestresultAvailable_NoConsentCloseAlertTitle" = "Wollen Sie wirklich abbrechen?";
 
 "ExposureSubmissionTestresultAvailable_NoConsentCloseAlertMessage" = "Ihre bisherigen Angaben werden nicht gespeichert.";
 
-"ExposureSubmissionTestresultAvailable_CloseAlertButtonCancel" = "Abbrechen";
+"ExposureSubmissionTestresultAvailable_CloseAlertButtonCancel" = "Ergebnis nicht anzeigen";
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:643]: Cool!

**Review by whiskey — APPROVED** [review]
Generally approving, although I hope the "Weiter"-issue will be solved as it's the case for `ExposureSubmissionTestresultAvailable_CloseAlertButtonContinue`

**Review by pascal-brause — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

**Review by SabineLoss — APPROVED** [review]

**Pull request merged**